packages/terra-form-input/src/terra-dev-site/doc/example/controlled/InputProgrammaticSet.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useRef } from 'react';
+import classNames from 'classnames/bind';
+import Button from 'terra-button';
+import Input from 'terra-form-input';
+import Styles from './examplestyles.scss';
```

Comment:
`Styles` should be lowercase here, `styles`

Reply:
updated here https://github.com/cerner/terra-core/pull/3135/commits/17a5bc400687a7f9b1b945eb45b51f42adbb27f6